examples/webgpu_postprocessing_3dlut.html

```diff
@@ -134,7 +135,7 @@
 
 				postProcessing = new THREE.PostProcessing( renderer );
 
-				const scenePass = pass( scene, camera );
+				const scenePass = pass( scene, camera, { samples: params.samples } );
```

Comment:
Do you mind picking another example which does not use a HDR workflow?

MSAA is not a good choice for HDR since depending on the sampled HDR texel value, the anti-aliasing effect get more or less disabled, see https://github.com/mrdoob/three.js/issues/26954#issuecomment-1759721712.

FX post-processing like FXAA or SMAA or the more high-quality SSAA/TAA are more appropriate for such scenes. We can add them step by step in the upcoming months. 

Even if the quality isn't that awesome, I would recommend to start with FXAA since it's always nice to have this performant technique as a option. `FXAAShader` can maybe be converted with TSL transpiler.

Reply:
Would `webgpu_loader_gltf.html` work then?

Reply:
I removed the example for now, maybe we can include it in the future into another example, I feel like `misc_controls_fly` would not really highlight the difference between the different amount of `samples` and creating a new example just for this seems too much.

Reply:
I think we could honor the samplers definied in `Renderer` for `pass()`, in case of `options=null`, with this logic `WebGPURenderer( { samples: params.samples } )` should work for all pass()

Reply:
Not sure if that's what you mean but basically check if the renderer is antialiased in `setup` of `PassNode` and if `this.options.samples === undefined` use `samples = 4` instead of `0`?

Reply:
Exactly, I think we could store samples in Renderer and reuse it in `setup( builder ) { ... builder.renderer }` moment to update the RenderTarget textures.